src/util/errors.js

```diff
@@ -136,6 +136,23 @@ export const isTransactionInitiateAmountTooLowError = error => {
   });
 };
 
+/**
+ * Check if the given API error (from `sdk.transaction.initiate()`) is
+ * due to other error in Stripe.
+ */
+export const isTransactionInitiateStripeError = error => {
+  if (error) {
+    return errorAPIErrors(error).reduce((messages, apiError) => {
+      const isPaymentFailedError =
+        apiError.status === 402 && apiError.code === ERROR_CODE_PAYMENT_FAILED;
+      let hasMessage = apiError && apiError.meta && apiError.meta.stripeMessage;
+
+      return isPaymentFailedError && hasMessage ? [...messages, hasMessage] : messages;
```

Comment:
This could be written as:
```js
const stripeMessageMaybe = isPaymentFailedError && apiError && apiError.meta
  ? [apiError.meta.stripeMessage]
  : [];
return [...messages, ...stripeMessageMaybe];
```